WordPress/Classes/ViewRelated/Post/PrepublishingViewController.swift

```diff
@@ -118,12 +150,16 @@ class PrepublishingViewController: UITableViewController {
 
         switch options[indexPath.row].id {
         case .tags:
+            cell.textLabel?.attributedText = rowTitle(rowID: .tags)
             configureTagCell(cell)
         case .visibility:
+            cell.textLabel?.attributedText = rowTitle(rowID: .visibility)
             configureVisibilityCell(cell)
         case .schedule:
+            cell.textLabel?.attributedText = rowTitle(rowID: .schedule)
             configureScheduleCell(cell)
         case .categories:
+            cell.textLabel?.attributedText = rowTitle(rowID: .categories)
```

Comment:
I think we could add a `subTitle` field to the `PrepublishingOption` struct and use a custom cell view, then we can 
just do:

```
cell.textLabel?.text = option.title
cell.subTitleLabel?.text = option.subTitle
```